### PR TITLE
Feature/judging portal

### PR DESCRIPTION
**Pull request by john-whiting**

### CODE REVIEW

File: client/src/components/ui/data-table/view-options.tsx
Comment (JarnaChao09) [inline client/src/components/ui/data-table/view-options.tsx:1]: is this needed or not?

Comment (JarnaChao09) [inline client/src/components/ui/data-table/view-options.tsx:1]: Is this file needed?

**Comment by john-whiting** [comment]
I removed the unnecessary comments/files.

**Review by hildebkg — APPROVED** [review]
